src/core/config.py

```
@@ -1,9 +1,57 @@
+SCORING_WEIGHTS: Dict[str, float] = {
+    "property_value": 0.3,
+    "location_score": 0.3,
+    "market_trend": 0.2,
+    "listing_age": 0.2
+}
+
+# Cache Configuration
+CACHE_CONFIG: Dict[str, Any] = {
+    "lead_scores": {"ttl": 3600},
+    "market_data": {"ttl": 7200},
+    "property_details": {"ttl": 86400}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)